onoff.js

```diff
@@ -195,11 +195,11 @@ class Gpio {
       });
     } else {
       return new Promise((resolve, reject) => {
-        fs.read(this._valueFd, this._readBuffer, 0, 1, 0, (err, bytes, buf) => {
-          if (err) {
-            reject(err);
+        this.read((error, value) => {
```

Comment:
All the code in onoff.js uses the name `err` rather than `error` for `Error` objects and I'd like to keep this consistent. Please rename `error` to `err`.